openstack/compute/v2/extensions/attachinterfaces/requests.go

```diff
@@ -11,3 +11,11 @@ func List(client *gophercloud.ServiceClient, serverID string) pagination.Pager {
 		return InterfacePage{pagination.SinglePageBase(r)}
 	})
 }
+
+// Get requests details on a single interface attachment by the server and port IDs.
+func Get(client *gophercloud.ServiceClient, serverID, portID string) (r GetResult) {
+	_, r.Err = client.Get(getInterfaceURL(client, serverID, portID), &r.Body, &gophercloud.RequestOpts{
+		OkCodes: []int{200, 203},
```

Comment:
Are you sure this returns both a 200 and 203?

Reply:
Not really. Thought that it is necessary, because `Get` for the regular server also contains that code:
https://github.com/gophercloud/gophercloud/blob/master/openstack/compute/v2/servers/requests.go#L305

Reply:
That's interesting. I have a feeling the `servers.Get` is incorrect. The string "203" doesn't exist anywhere in Nova. Would you mind removing this?